tradeexecutor/visual/custom_indicators.py

```diff
@@ -0,0 +1,37 @@
+"""Custom indicators for backtesting that are not found in pandas_ta."""
+
+import pandas as pd
+
+def on_balance_volume(_close_prices: pd.Series, volume: pd.Series) -> pd.Series:
```

Comment:
All Pythno functions should be named verb + subjective so that it is clear that is the action this function takes. I suggest renaming `calculate_on_balance_volume`